src/pages/home/report/ReportActionsView.js

```diff
@@ -353,7 +355,7 @@ class ReportActionsView extends React.Component {
 
         // Only mark as read if the report is open
         if (!this.props.isDrawerOpen) {
-            Report.updateLastReadActionID(this.props.reportID);
+            Report.readNewestAction(this.props.reportID);
```

Comment:
As I mentioned in the other comment, I think we can do away with this method once `OpenReport` is implemented

Reply:
I'm less certain it can be removed as the "new marker" is still getting updated here. But I think the design for that is all wrong and have been looking into how to improve it.

Reply:
It's a bit more complicated than that because the "marker position" is currently tracked as separate property in Onyx and can also be a different value than the last read.